service/prometheus/group.go

```diff
@@ -1,47 +0,0 @@
-package prometheus
```

Comment:
Note: `group.go` was renamed to `filter.go`, and `GroupServices` was removed, but I can't get the diff to look nice :D